lib/appsignal_plug.ex

```diff
@@ -45,14 +45,16 @@ defmodule Appsignal.Plug do
 
       def call(conn, opts) do
         Appsignal.instrument(fn span ->
-          @span.set_namespace(span, "http_request")
+          _ = @span.set_namespace(span, "http_request")
 
           try do
             super(conn, opts)
           catch
             kind, reason ->
               stack = __STACKTRACE__
 
+              _ = span
```

Comment:
I'm not quite sure what this does. We call span on line 58, right? What does it do by calling it again? 🤔

Reply:
Ha, good catch! That was a merge fail. Fixed!